web/src/message_summary.ts

```diff
@@ -44,11 +44,14 @@ export function get_narrow_summary(channel_id: number, topic_name: string): void
     }
     dialog_widget.launch({
         text_heading: display_topic_name,
-        html_body: "",
+        html_body: "<div></div>", // Placeholder, will be replaced in post_render.
```

Comment:
This isn't required for this PR, but I assume what we actually want here is a loading indicator?